internal/commands/configure/configure.go

```diff
@@ -59,6 +59,108 @@ var discordEmojiRegex = regexp.MustCompile(`<(a)?:([^<>:]+):([0-9]+)>`)
 
 var unicodeEmojiRegex = regexp.MustCompile(`[\x{00A0}-\x{1F9EF}]`)
 
+// configureValue configures a generic value to be used elsewhere in the code
+// it's for values that might not make sense to have in a separate category (like channels)
+func (c *Command) configureValue(s *discordgo.Session, m *discordgo.MessageCreate, parts []string) {
+	validKeys := map[string]string{
+		"pajbot_host": "The host used for any pajbot API requests and commands (like $points). Example value: forsen.tv. Assumes https schema and standard api path.",
+	}
+
+	const usage = "usage: $configure value (reset/set/get/keys) [key [value]]"
+	if len(parts) < 2 {
+		s.ChannelMessageSend(m.ChannelID, usage)
+		return
+	}
+
+	if parts[1] == "keys" {
+		var chunks []string
+		for validKey, keyDescription := range validKeys {
+			chunks = append(chunks, fmt.Sprintf("%s: %s\n", validKey, keyDescription))
+		}
+		utils.SendChunks("Valid keys:\n```", "```", chunks, m.ChannelID, s)
+
+		return
+	}
+
+	if len(parts) < 3 {
+		s.ChannelMessageSend(m.ChannelID, usage)
+		return
+	}
+
+	key := parts[2]
+
+	if _, ok := validKeys[key]; !ok {
+		const f = "%s, %s is not a valid key. use $configure value keys to see valid keys"
+		r := fmt.Sprintf(f, m.Author.Mention(), key)
+		s.ChannelMessageSend(m.ChannelID, r)
+		return
+	}
+
+	dbKey := fmt.Sprintf("value:%s", key)
+
+	oldValue := serverconfig.Get(m.GuildID, dbKey)
+
+	switch parts[1] {
+	case "reset":
+		err := serverconfig.Remove(commands.SQLClient, m.GuildID, dbKey)
+		if err != nil {
+			fmt.Println("Error removing value:", err)
+			return
+		}
+
+		const f = "%s, key %s reset. old value was %s"
+		r := fmt.Sprintf(f, m.Author.Mention(), key, oldValue)
+		s.ChannelMessageSend(m.ChannelID, r)
```

Comment:
This seems to be appended even when there was no original value. Are we able to not post this portion of the message if there was no `oldValue`?

![image](https://user-images.githubusercontent.com/12804673/108475832-fa248100-72dc-11eb-813d-429458b5d9f3.png)

Reply:
Fixed it in [`3162b66` (#46)](https://github.com/pajbot/pajbot2-discord/pull/46/commits/3162b667ecad861605a2aa6cb7449e9d2f80d99c) :+1: